.github/workflows/build.yaml

```diff
@@ -620,19 +629,46 @@ jobs:
         run: |
           make -C operator/ bundle bundle-build
 
+      - name: Operator unit tests
+        if: matrix.arch == 'amd64'
+        run: |
+          GOOS=linux scripts/lib.sh retry 6 true make -C operator/ test
```

Comment:
1. Do we need the `GOOS` setting here? 🤔 We're not cross-building so it will just use the native os, no?
2. I don't think 6 attempt (or any retries for that matter) are needed for unit tests. WDYT?